packages/forms-web-app/src/pages/projects/relevant-representations/representations.js

```diff
@@ -70,7 +71,8 @@ exports.getRepresentations = async (req, res) => {
 		pageOptions,
 		searchTerm,
 		queryUrl,
-		commentsTypeFilterItems
+		commentsTypeFilterItems,
+		showReps: isDateAfterTodaysDate(applicationResponse.DateRRepAppearOnWebsite)
```

Comment:
I think we may have to handle the case where the date is not set.  I think .DateRRepAppearOnWebsite  gets set just before they publish the reps so can this handle null or not set?

Reply:
date not set will result in an invalid date and be false.

Reply:
i'll add a test for the case.